java/client/src/main/java/glide/api/BaseClient.java

```diff
@@ -28,6 +30,59 @@ protected static Object handleObjectResponse(Response response) {
         return (new BaseCommandResponseResolver(RedisValueResolver::valueFromPointer)).apply(response);
     }
 
+    /**
+     * Check for errors in the Response and return null Throws an error if an unexpected value is
+     * returned
+     *
+     * @return null if the response is empty
```

Comment:
Should we use the @throws tag here?
```suggestion
     * Check for errors in the Response and return null. 
     *
     * @return null if the response is empty
     * @throws RedisException if an unexpected (non-null) value is returned
```

Reply:
+1